app/cdap/components/GridTable/index.tsx

```diff
@@ -52,21 +58,23 @@ export default function GridTable() {
 
   const classes = useStyles();
 
-  const [loading, setLoading] = useState(false);
+  const [loading, setLoading] = useState<boolean>(false);
   const [headersNamesList, setHeadersNamesList] = useState([] as IHeaderNamesList[]);
-  const [rowsDataList, setRowsDataList] = useState([]);
+  const [rowsDataList, setRowsDataList] = useState([] as IRowData[]);
```

Comment:
const [rowsDataList, setRowsDataList] = useState([] as IRowData[]);

should be 
const [rowsDataList, setRowsDataList] = useState<IRowData[]>([]);

Reply:
Please make sure all types are given like this